docs/monoDrive_home/Simulator-Commands.md

```diff
@@ -0,0 +1,365 @@
+# Simulator API Commands
+
+## Get Simulator Version
+Retrieves the current version of the simulator and of the simulator API with the commands documented on this page.
+
+**command ID**:  "GetVersion"
+
+**command data**:
+```json
+{	
+}
+```
+**response**: (string)
+
+`simulator_version: 1.12, api_version: 4.0`
+
+
+## Simulator Configuration
+Used to configure a simulator session. This command includes the settings associated with the simulator session following the execution of the command and applies to all future sessions until the command is run again. The "pys_materials" settings affect the returns from lidar and radar sensors for the list of known materials in the scene.
```

Comment:
You can save the `phys_materials` description for the documentation below about those parameters.